Customization/T200/PhoneRepairShop_Code/PhoneRepairShop_Code.sln

```diff
@@ -1,10 +1,10 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
-VisualStudioVersion = 16.0.31019.35
+VisualStudioVersion = 16.0.31727.386
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{E24C65DC-7377-472B-9ABA-BC803B73C61A}") = "PhoneRepairShop", "..\..\..\", "{407B75C1-DF84-479F-BD54-F5A5D7012034}"
 	ProjectSection(WebsiteProperties) = preProject
-		TargetFrameworkMoniker = ".NETFramework,Version%3Dv4.8"
+		TargetFrameworkMoniker = ".NETFramework,Version%3Dv4.7.1"
```

Comment:
Acumatica ERP requires .net framework 4.8 as far as I know. These change should be reverted.